streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamGlobalKTableJoinTest.java

```diff
@@ -145,8 +146,8 @@ public void shouldNotJoinOnGlobalTableUpdates() {
         // push all four items to the primary stream. this should produce two items.
 
         pushToStream(4, "X", true);
-        processor.checkAndClearProcessResult("0:X0,FKey0+Y0 (ts: 0)", "1:X1,FKey1+Y1 (ts: 1)");
-
```

Comment:
Why remove the empty line? It make it harder to read the code, as logical blocks are separated by blank lines atm. (similar below)

Reply:
ok done